code/src/components/Game.js

```diff
@@ -0,0 +1,17 @@
+import React from "react"
+import { useSelector } from "react-redux"
+import { Start } from "./Start"
+import { GameBoard } from "./GameBoard"
+import { LoadingIndicator } from "./LoadingIndicator"
+
+export const Game = () => {
+  const loading = useSelector((store) => store.ui.loading)
+  const currentPosition = useSelector((store) => store.game.currentPosition)
+
+  return (
+    <>
+      {loading && <LoadingIndicator />}
```

Comment:
Good thing you never used the setTimeout, we get to see the loading indicator long enough anyway.